pages.de/common/age.md

```diff
@@ -0,0 +1,20 @@
+# age
+
+> Ein einfaches, modernes und sicheres Dateiverschlüsselungswerkzeug.
+> Mehr Informationen: <https://age-encryption.org>.
+
+- Eine verschlüsselte Datei, die mit einer Passphrase entschlüsselt werden kann, generieren:
+
+`age -p -o {{/pfad/zu/verschluesselter_datei}} {{/pfad/zu/unverschluesselter_datei}}`
+
+- Ein Schlüsselpaar generieren, wobei der öffentlich und private Schlüssel in einer (unverschlüsselten) Schlüsseldatei gespeichert werden und der öffentliche Schlüssel zu stdout ausgegeben wird:
+
+`age-keygen -o {{/pfad/zu/schluesseldatei}}`
+
+- Eine Datei zu einem oder mehreren öffentlichen Schlüsseln, die als Zeichenketten eingegeben werden, verschlüsseln:
+
+`age -o {{/pfad/zu/verschluesselter_datei}} -r {{oeffentlicher_schluessel_1}} -r {{oeffentlicher_schluessel_2}} {{/pfad/zu/unverschluesselter_datei}}`
+
+- Eine Datei zu einem oder mehreren öffentlichen Schlüsseln, die in einer Empfängerdatei angegeben sind, in welcher jeder öffentliche Schlüssel mit einer Neuzeile getrennt ist und Zeilen, die mit `#` anfangen, ignoriert werden, verschlüsseln:
```

Comment:
```suggestion
- Eine Datei zu einem oder mehreren öffentlichen Schlüsseln, die in einer Empfängerdatei angegeben sind, verschlüsseln:
```
I think we can delete that part, since it doesn't seem really neccessary.